src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
```

Comment:
The class name 'RightTriangle' is a noun, which is good practice, but it seems like there is an empty line at the beginning of the class definition that should be removed.